### PR TITLE
Adds testing to cli-plugins-repo PRs

### PR DESCRIPTION
**Pull request by jdgonzaleza**
Currently one of the pain points of reviewing
and approving plugins PR is the fact that there's
a possibility that authors forgot to run
the sort command before publishing the PR.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Run Tests
+        run: |
+          set -ex
+
+          go install github.com/onsi/ginkgo/ginkgo@latest
+
+          BINARY_VALIDATION=false bin/test
```
Comment (a-b) [inline .github/workflows/test.yml:24]: What do you think if we would configure `BINARY_VALIDATION` on a step level?
https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsstepsenv

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+on:
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  test:
+    name: Running Unit Tests
+    runs-on: ubuntu-latest
+    container: cfcli/cli-base:latest
```
Comment (a-b) [inline .github/workflows/test.yml:12]: I'm wondering if using a generic golang base image would future prof us by reducing one dependency.
I'm thinking about [this](https://github.com/cloudfoundry/cli/blob/5b57d4f0afdcdcda18adc1805d269c4e9164d3c7/.github/workflows/units.yml#L42-L45)

Comment (a-b) [reply .github/workflows/test.yml:12]: In the past, cli-base was pretty much the only option, but with GHA we have more choices.

**Review by ccjaimes — APPROVED** [review]
Looks like a good test to me!

**Pull request merged**